config/crd/bases/executor.testkube.io_executors.yaml

```diff
@@ -61,6 +72,18 @@ spec:
               image:
                 description: Image for kube-job
                 type: string
+              imagePullSecrets:
+                description: container executor default image pull secrets
+                items:
+                  description: LocalObjectReference contains enough information to
+                    let you locate the referenced object inside the same namespace.
+                  properties:
+                    name:
+                      description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                        TODO: Add other useful fields. apiVersion, kind, uid?'
+                      type: string
```

Comment:
is this a reference to a secret or just a secret itself (we can't store secrets in plain forms in CRD)?

Reply:
ok, will need to add some docs regarding this to show how to use it